containers/src/Kokkos_DynRankView.hpp

```diff
@@ -1958,7 +1958,7 @@ inline auto create_mirror(const DynRankView<T, P...>& src,
     return dst_type(prop_copy,
                     Impl::reconstructLayout(src.layout(), src.rank()));
   }
-#if defined KOKKOS_COMPILER_INTEL
+#if defined(KOKKOS_COMPILER_INTEL) || defined(KOKKOS_COMPILER_NVCC)
```

Comment:
Should `&& defined(KOKKOS_COMPILER_NVCC >= 1130))`  be added here as well?

Reply:
Thanks, fixed with 7e974b93cc25b43c778e4dd4eb8b8439be0b2ae7